cypress/e2e/upload-download.cy.ts

```diff
@@ -1,10 +1,17 @@
-import {Upload} from "../page/index";
+import {
+  Upload,
+  Download,
 
-describe("uploading a file test", () => {
+}
+  from "../page/index";
```

Comment:
fix the indentation:
```suggestion
from "../page/index";
```